contracts/base/ModuleManager.sol

```diff
@@ -204,4 +245,31 @@ abstract contract ModuleManager is SelfAuthorized, Executor, GuardManager, IModu
         /* solhint-enable no-inline-assembly */
         return size > 0;
     }
+
+    /**
+     * @inheritdoc IModuleManager
+     */
+    function setModuleGuard(address moduleGuard) external override authorized {
+        if (moduleGuard != address(0)) {
+            require(IModuleGuard(moduleGuard).supportsInterface(type(IModuleGuard).interfaceId), "GS301");
```

Comment:
Just wanted to doublecheck that this error code isn't used anywhere.

Reply:
Also, this should not use `require` - @remedcu added code that is much more space efficient for reverts: `revertWithError("GS301");`

Reply:
I had checked it already.
<img width="426" alt="Screenshot 2024-05-07 at 12 22 05" src="https://github.com/safe-global/safe-smart-account/assets/16358982/96b2ed01-b10e-4435-a3ab-df1b01a59f04">

Reply:
https://github.com/safe-global/safe-smart-account/pull/758/commits/dd78fdcb8b40a59edb014c4ee454438dee26fb54